operator_ui/src/components/Jobs/RecentlyCreated.js

```diff
@@ -55,7 +55,7 @@ const RecentlyCreated = ({ jobs, classes }) => {
                     component="span"
                     color="primary"
                   >
-                    {j.id}
+                    {j.name ? j.name : j.id}
```

Comment:
Job names are not unique, it may be useful to always include the ID here

Reply:
There is an id right in front of it